packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js

```diff
@@ -23,14 +24,31 @@ export default class DedicatedServerDashboard {
     this.$state = $state;
     this.$stateParams = $stateParams;
     this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
     this.constants = constants;
     this.DedicatedServerFeatureAvailability = DedicatedServerFeatureAvailability;
     this.Server = Server;
   }
 
+  loadMessages() {
+    this.CucCloudMessage.unSubscribe('app.dedicated.server.dashboard');
```

Comment:
Why not use `Alerter` as it's been done elsewhere ?

Reply:
this part of code is a miss, i use alerter in practice